pages/all-about.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>All About DIK-DIKS!</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/all-about.css">
+  <link href="https://fonts.googleapis.com/css2?family=Gloria+Hallelujah&display=swap" rel="stylesheet">
+
+  <style>
+  .dikdik-banner-container {
+      position: relative;
+      z-index: 1000;
+      width: 100%;
+      background-color: #f5e9d2; 
+  }
+  </style>
+
+</head>
+
+<body>
+  <div class="page-wrapper">
+    <header>
+      <h1>All About Dik-Diks!</h1>
+      <nav>
+        <ul>
+          <li><a href="../index.html">Home</a></li>
+          <li><a href="all-about.html">All About Dik-Diks</a></li>
+          <li><a href="fun-facts.html">Fun Facts</a></li>
+          <li><a href="gallery.html">Gallery</a></li>
+        </ul>
+      </nav>
+    </header>
+
+  <div class="dikdik-banner-container"> 
+    <div class="dikdik-banner">
+        <img src="../assets/dikdik-stare.jpg" alt="Close-up of a dik-dik in natural habitat">
+    </div>
+  </div>
+
+  <div class="facts-heading">
+  <h2>Interesting Facts About Dik-Diks:</h2>
+  </div>
+
+    <main class="grid-container">
+
+      <div class="flip-card">
+        <div class="flip-inner">
+          <div class="flip-front">SCIENTIFIC NAME</div>
+          <div class="flip-back">MADOQUA KIRKII</div>
+        </div>
+      </div>
+
+      <div class="flip-card">
+        <div class="flip-inner">
+          <div class="flip-front">WEIGHT</div>
+          <div class="flip-back">
+            <ul class="fact-list">
```

Comment:
Is there a more semantic element for the `div` tag on line 58 for "Weight"? Could we use a header tag?

Also, do we need this fourth `div` here? What's it do?

Could we put the class "flip-back" onto the unordered list element?

```suggestion
          <h3 class="flip-front">WEIGHT</h3>
            <ul class="fact-list flip-back">
```

Looking at each of your flip cards on this page, are there more semantic elements you could use? Do you need all the `div` elements for each flip cards?

If it's not used for semantics and it's not necessary for formatting (I think this is true because I removed the div and ran your code to see if it affected the layout and it didn't) then we should prefer not to bring in unnecessary elements.